src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -20,6 +22,7 @@ class RunWorkflowTests : SeleniumTest()
         JooqContext().use {
             insertUserAndGroup(it, "test.user@example.com")
             insertWorkflow("test.user@example.com", "newkey", "workflow1")
+            insertReport("test_report", "version1")
```

Comment:
Is this report needed by your test? Not obvious why it has been added.

Reply:
I think the thinking was that there are reports in `workflowRunStatus` that needed to be supplied. I'll remove it and see what happens.

Reply:
Ah ok. If it is necessary that's obviously fine, but if so then I would definitely recommend moving the `insertReport()` into your test, and add an appropriate comment. Adding to the global setup generally isn't ideal unless needed for all tests, because it changes the environment in which all current (and future) tests run.